src/routes/github/webhook/webhook-receiver-post.ts

```diff
@@ -56,6 +58,21 @@ const webhookRouter = (context: WebhookContext) => {
 				GithubWebhookMiddleware(issueWebhookHandler)(context);
 			}
 			break;
+		case "pull_request":
+			if (context.action === "opened" || context.action === "closed"
+				|| context.action === "reopened" || context.action === "edited") {
+				GithubWebhookMiddleware(pullRequestWebhookHandler)(context);
```

Comment:
/nit do you expect this list to grow?
I'm quite fond of having `const VALID_PR_ACTIONS = [""opened","reopened","closed","edited"]`
Then you can just `VALID_PR_ACTIONS.includes(context.action)`

might be overkill tho

Reply:
It is final as of now.

Reply:
personally, I'd still like to see josh's suggestion.  It's much cleaner

Reply:
Done!!